projectsmember/Pages/CooperationRequest/Profiles/HamidForoughnia.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>H.Foroughnia's Online Resume</title>
+    <link href="../../../Content/Bootstrap/bootstrap.min.css" />
```

Comment:
Now that you are using the bootstrap 5.1.3, you can remove this line, no need for local versions.